sources/sources.bib

```diff
@@ -25,3 +25,16 @@ @article{sample_ref
   publisher={Springer}
 }
 
+@inproceedings{viola_jones,
+  author = {Viola, Paul and Jones, Michael},
+  title = {Remember to Smile: Design of a Mobile Affective Technology to Help Promote Individual Happiness through Smiling},
+  year = {2017},
+}
+
+@inproceedings{opencv_cascade_classifier,
```

Comment:
When citing online sources, you usually have to specify when you retrieved it, e. g. "Retrieved September 05, 2020" due to websites being rather volatile.